src/controllers/ProfileUpdateController.js

```diff
@@ -0,0 +1,139 @@
+   * @static
+   * @param {object} req client request
+   * @param {object} res server response
+   *  @param {object} next next middleware
+   * @returns {object} server response object
+   * @memberof ProfileController
+   */
+  static async signInUser(req, res) {
+    const { email, password } = req.body;
+    try {
+      const user = await Users.findOne({ where: { email } });
+      const match = await bcrypt.compare(password, user.password);
+      if (match) {
+        ResponseHandler.success(res, user, 200);
+      }
+      ResponseHandler.error(res, 'login details do not match', 401);
+    } catch (error) {
+      ResponseHandler.error(res, error, 404);
+    }
+  }
+
+  /**
+   *
+   * @description method that updates user's profile settings
+   * @static
+   * @param {object} req client request
+   * @param {object} res server response
+   * @returns {object} server response object
+   * @memberof ProfileController
+   */
+  static async updateProfile(req, res) {
+    const userId = parseInt(req.params.id, 10);
+    try {
+      const user = await Users.findOne({ where: { id: userId } });
+      try {
+        const oldDate = user.birthdate;
+        await user.update({
+          firstName: req.body.firstName || user.firstName,
+          lastName: req.body.lastName || user.lastName,
+          password: req.body.password || user.password,
+          gender: req.body.gender || user.gender,
+          birthdate: returnDate(req.body.birthdate, oldDate),
+          preferredLanguage:
+            req.body.preferredLanguage || user.preferredLanguage,
+          preferredCurrency:
+            req.body.preferredCurrency || user.preferredCurrency,
+          city: req.body.city || user.city,
+          state: req.body.state || user.state,
+          zip: req.body.zip || user.zip,
+          country: req.body.country || user.country,
+          role: req.body.role || user.role,
+          department: req.body.department || user.department,
+          lineManager: req.body.lineManager || user.lineManager
+        });
+      } catch (error) {
+        ResponseHandler.error(res, 'error', 400);
+      }
+      ResponseHandler.success(res, user);
+      return res.status(200).json({
+        status: 'success',
+        data: user
+      });
+    } catch (err) {
+      ResponseHandler.error(res, 'no user with that id was found', 404);
+    }
+  }
+
+  /**
+   *
+   * @description method that gets current user's settings
+   * @static
+   * @param {object} req client request
+   * @param {object} res server response
+   * @returns {object} server response object
+   * @memberof ProfileController
+   */
+  static async getProfile(req, res) {
+    const userId = parseInt(req.params.id, 10);
+    try {
+      const user = await Users.findOne({ where: { id: userId } });
+      ResponseHandler.success(res, user);
+    } catch (error) {
+      ResponseHandler.error(res, 'No user with that id was found', 404);
+    }
+  }
+}
```

Comment:
I see no need for creating a `ProfileUpdateController.js`. if at all you want to implement your controller in a separate file, you name the file `userController.js`. Also, check the way I import models into controllers. Also, check the way I import files from the utility folder. you don't just create your own style, you follow the style the codebase is using.